sample/ApiSamples/Program.cs

```diff
@@ -20,6 +24,47 @@ private static void Main(string[] args)
 			Console.ReadKey();
 		}
 
+		public static void SampleSpamWithCustomContext()
+		{
+			Agent.Tracer.CaptureTransaction("SampleTransaction", "SampleTransactionType", transaction =>
+			{
+				transaction.CaptureSpan("SampleSpan", "SampleSpanType", span =>
+				{
+					span.Context.Db = new Database
+					{
+						Statement = "Select * from MyTable",
+						Type = "MSSQL"
```

Comment:
There is, but those are not the same: https://github.com/elastic/apm/issues/8 

I think it's a good idea to use the constant in the sample code -> changed it.

Reply:
Update: yeah, that should be `sql`....

Reply:
Update2: solved it with constants on the `Database` class. `ApiConstants` contains constants for the interfaces (since those can't contain consts), the remaining constants can be added to public classes - e.g. db related constants to the `Database` class.